trace/nox.py

```diff
@@ -39,12 +40,43 @@ def unit_tests(session, python_version):
         '--cov-config=.coveragerc',
         '--cov-report=',
         '--cov-fail-under=97',
-        'tests/',
+        'tests/unit/',
         *session.posargs
     )
 
 
 @nox.session
+@nox.parametrize('python_version', ['2.7', '3.5'])
+def system_tests(session, python_version):
+    """Run the system test suite."""
+
+    # Sanity check: Only run system tests if the environment variable is set.
+    if not os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', ''):
+        session.skip('Credentials must be set via environment variable.')
+
+    # Run the system tests against latest Python 2 and Python 3 only.
+    session.interpreter = 'python{}'.format(python_version)
+
+    # Set the virtualenv dirname.
+    session.virtualenv_dirname = 'sys-' + python_version
+
+    # Install all test dependencies, then install this package into the
+    # virutalenv's dist-packages.
+    session.install('-r', 'requirements-test.txt')
+    session.install('.')
+
+    # Run py.test against the system tests.
+    session.run(
+        'py.test',
+        '-vvv',
+        '-s',
+        'tests/system/',
+        *session.posargs,
+        success_codes=range(0, 100)
```

Comment:
What does this do?  It seems like it ignores all error codes in the [1,99] range?  Can we reduce this list, and document the expected error codes?

Reply:
Removed. Will add documentation after several observations of the system tests failure(if any).